README.md

```diff
@@ -17,7 +17,7 @@ This Operator is in the early stages of implementation. For the time being, plea
    ```
```

Comment:
Please consider revising the introduction to:

"The `ExternalDNS` Operator allows you to deploy and manage ExternalDNS, a cluster-internal DNS server in Kubernetes which makes Kubernetes resources discoverable through public DNS servers.
**Note**: This Operator is in the early stages of implementation. For more information, see
ExternalDNS Operator OpenShift Enhancement Proposal."

Reply:
Ok will do